content/hi/docs/reference/glossary/api-group.md

```diff
@@ -0,0 +1,19 @@
+---
+title: API समूह
+id: api-group
+date: 2019-09-02
+full_link: /docs/concepts/overview/kubernetes-api/#api-groups-and-versioning
+short_description: >
+  कुबेरनेट्स API में संबंधित पथों का एक समूह।
+
+aka:
+tags:
+- fundamental
+- architecture
+---
+कुबेरनेट्स API में संबंधित पथों का एक समूह।
+
+<!--more-->
+आप अपने API सर्वर के कॉन्फ़िगरेशन को बदलकर प्रत्येक API समूह को सक्षम या अक्षम कर सकते हैं। आप विशिष्ट संसाधनों के लिए पथ अक्षम या सक्षम भी कर सकते हैं। API समूह कुबेरनेट्स API का विस्तार करना आसान बनाता है। API समूह एक REST पथ में और एक क्रमबद्ध वस्तु के `apiVersion` फ़ील्ड में निर्दिष्ट है।
+
+* अधिक जानकारी के लिए पढ़ें [API समूह](/docs/concepts/overview/kubernetes-api/#api-groups-and-versioning)
```

Comment:
Just a small nit, otherwise changes LGTM
```suggestion
* अधिक जानकारी के लिए [API समूह](/docs/concepts/overview/kubernetes-api/#api-groups-and-versioning) पढ़ें।
```

Reply:
Thanks @Babapool for suggesting this